packages/core/src/hosted-form/iframe-content/hosted-input-validator.ts

```diff
@@ -199,4 +199,8 @@ export default class HostedInputValidator {
                 return type;
         }
     }
+
+    private _isValidErrorType(error: any): error is { name: 'ValidationError' } {
+        return error.name === 'ValidationError';
```

Comment:
```suggestion
        return name in error;
```

Reply:
Thanks for the feedback.
I have updated the typeguard this way.